app/views/admin/tags/edit.html.erb

```diff
@@ -5,7 +5,7 @@
   </div>
 
   <div class="ml-auto">
-    <a href="/t/<%= @tag.name %>">View</a>
+    <a href="<%= tag_path(@tag.name) %>" class="crayons-btn crayons-btn--outlined" target="_blank" rel="noopener noreferer">View</a>
```

Comment:
I love the use of `<%= tag_path(@tag.name) %>` in place of `/t/<%= @tag.name %>` here!